apps/public-docsite/src/components/Site/Site.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
```

Comment:
This file is where the title is updated for the actual public site (developer.microsoft.com/fluentui)